src/main/java/frc/robot/RobotContainer.java

```diff
@@ -2,23 +2,29 @@
 
 import edu.wpi.first.wpilibj2.command.button.Trigger;
 import frc.robot.Constants.OperatorConstants;
-import frc.robot.commands.SetAxisCommand;
 import frc.robot.subsystems.ArmSystem;
+import frc.robot.subsystems.IntakeSystem;
 import frc.robot.subsystems.ShooterSystem;
+import frc.robot.commands.VisionAutoAlignCommand;
 import frc.robot.subsystems.SwerveDriveSystem;
+import frc.robot.subsystems.VisionSystem;
 import frc.robot.util.AppliedController;
-import frc.robot.util.Coords;
 
 /**
  * RobotContainer.
  */
 public class RobotContainer {
     private final AppliedController m_driveController = new AppliedController(
             OperatorConstants.driveControllerPort);
+    private final AppliedController m_armController = new AppliedController(
+            OperatorConstants.armControllerPort);
 
-    private final SwerveDriveSystem m_swerveDrive = new SwerveDriveSystem(m_driveController);
-    private final ShooterSystem m_shooter = new ShooterSystem();
-    private final ArmSystem m_ArmAndIntakeSystem = new ArmSystem();
+    private SwerveDriveSystem m_swerveDrive = new SwerveDriveSystem(m_driveController);
+    private VisionSystem m_visionSystem = new VisionSystem();
```

Comment:
Technically, it is safer to have them as final variables so they cannot be changed which can mess up the robot. This may be an issue in the future. This will be fixed on fix-issues branch with everything as final.